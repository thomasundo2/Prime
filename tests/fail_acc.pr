int main()
{
    /* access an integer id */
    int i;
    i = 1;
    printl(i.x);
}