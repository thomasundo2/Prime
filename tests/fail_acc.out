Fatal error: exception Failure("cannot access type: int in i.x")
